app/helpers.js

```diff
@@ -57,4 +57,13 @@ Helpers.getDateFromNow = function (timestamp) {
   return moment(timestamp, 'X').fromNow();
 };
 
+/**
+ * Escapes any characters that have special meaning in JQuery selectors.
```

Comment:
We haven't used comments like this before. I'm not super convinced we need them. Could you take these out. 
We can have a bigger discussion around comments like this and then look to add them in multiple places.